### ISSUE TITLE
Allow multi-factor selection in non-dict factor selection mode

### ISSUE THREAD

**Issue by modantailleur** [mathieulagrange/doce#217]
Could you allow to use a command like:

` python main.py -s plan1/dataset=A&B+learning_rate=-3&-4`

to select multiple factors in this non-dict selection mode ?

**Comment by mathieulagrange** [comment]
& is shell token: https://www.oreilly.com/library/view/learning-the-bash/1565923472/ch01s09.html

I can try with ,

**Comment by mathieulagrange** [comment]
Here is a proposal:

python3 demo.py -f -s nn_type=cnn+lstm,dropout=0:nn_type=cnn,dropout=1
dropout=0+learning_rate=0.001+n_layers=2+nn_type=cnn
dropout=0+learning_rate=0.0001+n_layers=2+nn_type=cnn
dropout=0+learning_rate=0.00001+n_layers=2+nn_type=cnn
dropout=0+learning_rate=0.001+n_layers=5+nn_type=cnn
dropout=0+learning_rate=0.0001+n_layers=5+nn_type=cnn
dropout=0+learning_rate=0.00001+n_layers=5+nn_type=cnn
dropout=0+learning_rate=0.001+n_layers=8+nn_type=cnn
dropout=0+learning_rate=0.0001+n_layers=8+nn_type=cnn
dropout=0+learning_rate=0.00001+n_layers=8+nn_type=cnn
dropout=0+learning_rate=0.001+n_layers=2+nn_type=lstm
dropout=0+learning_rate=0.0001+n_layers=2+nn_type=lstm
dropout=0+learning_rate=0.00001+n_layers=2+nn_type=lstm
dropout=0+learning_rate=0.001+n_layers=5+nn_type=lstm
dropout=0+learning_rate=0.0001+n_layers=5+nn_type=lstm
dropout=0+learning_rate=0.00001+n_layers=5+nn_type=lstm
dropout=0+learning_rate=0.001+n_layers=8+nn_type=lstm
dropout=0+learning_rate=0.0001+n_layers=8+nn_type=lstm
dropout=0+learning_rate=0.00001+n_layers=8+nn_type=lstm
dropout=1+learning_rate=0.001+n_layers=2+nn_type=cnn
dropout=1+learning_rate=0.0001+n_layers=2+nn_type=cnn
dropout=1+learning_rate=0.00001+n_layers=2+nn_type=cnn
dropout=1+learning_rate=0.001+n_layers=5+nn_type=cnn
dropout=1+learning_rate=0.0001+n_layers=5+nn_type=cnn
dropout=1+learning_rate=0.00001+n_layers=5+nn_type=cnn
dropout=1+learning_rate=0.001+n_layers=8+nn_type=cnn
dropout=1+learning_rate=0.0001+n_layers=8+nn_type=cnn
dropout=1+learning_rate=0.00001+n_layers=8+nn_type=cnn

**Comment by mathieulagrange** [comment]
can you test and provide feedback ?

**Comment by modantailleur** [comment]
It works well
